### PR TITLE
Make a Gusto webpage

### PR DESCRIPTION
**Pull request by colinjcotter**
I started this pull request early, so that people can comment on what is going on and fix if they are able/willing.

The idea is to clone the Firedrake docs to start making docs for dcore, shortly to be known as Gusto.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.. title:: The Firedrake project
```
Comment (dham) [inline docs/index.rst]: Probably want to fix this ;)

File: docs/teamgrid.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
+..
```
Comment (dham) [inline docs/teamgrid.rst]: Don't commit this file. Committing autogenerated files just causes problems later.

**Comment by dham** [comment]
It's a little hard to be precise in commenting becase the one big commit both copies Firedrake and contains all the edits. It's therefore hard to see what was changed.

**Comment by colinjcotter** [comment]
OK, can you tell me how to build the website? Then I can check these things
myself.

On 3 June 2016 at 16:45, David A. Ham <notifications@github.com> wrote:

> It's a little hard to be precise in commenting becase the one big commit
> both copies Firedrake and contains all the edits. It's therefore hard to
> see what was changed.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/firedrakeproject/dcore/pull/41#issuecomment-223615295>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF0q8hS17a0wwXi-uul5V-7C-_y6yrmNks5qIEv9gaJpZM4ItpEj>
> .
>



-- 
http://www.imperial.ac.uk/people/colin.cotter

www.cambridge.org/9781107663916

**Comment by wence-** [comment]
I'd remove (in the sense of never having committed them) most of the image files that are specific to the firedrake website (e.g. the firedrake cake, the fenics15 banner, etc...)

**Comment by dham** [comment]
you can't currently build the website because your directory structure is completely shot

I'll fix it

**Comment by wence-** [comment]
You've managed to commit a load of crud, but not the crucial sphinx files which tell it how to build the documentation from the rst.

I recommend starting by running `sphinx-quickstart` in a clean tree.

**Comment by wence-** [comment]
I think the banner logo looks a bit big, and probably wants to be floated right.

Andrew is at Bath, so the team page is wrong, no?

**Comment by colinjcotter** [comment]
So, the Firedrake page also has a few former Imperial people on there...

On 7 June 2016 at 09:08, Lawrence Mitchell <notifications@github.com> wrote:

> I think the banner logo looks a bit big, and probably wants to be floated
> right.
>
> Andrew is at Bath, so the team page is wrong, no?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/firedrakeproject/dcore/pull/41#issuecomment-224209659>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF0q8mAOURWPdpOY7gNK5tYWY7GoWBE4ks5qJSb-gaJpZM4ItpEj>
> .
>



-- 
http://www.imperial.ac.uk/people/colin.cotter

www.cambridge.org/9781107663916

**Comment by colinjcotter** [comment]
OK, merging.

**Pull request merged**